_overviews/scala3-book/methods-most.md

```diff
@@ -359,30 +624,46 @@ Their main purpose is to let you add new functionality to closed classes.
 As shown in that section, imagine that you have a `Circle` class, but you can’t change its source code.
 For instance, it may be defined like this in a third-party library:
 
+{% tabs method_23 %}
+{% tab 'Scala 2 and 3' for=method_23 %}
+
 ```scala
 case class Circle(x: Double, y: Double, radius: Double)
 ```
 
+{% endtab %}
+{% endtabs %}
+
 When you want to add methods to this class, you can define them as extension methods, like this:
 
+{% tabs method_24 %}
+{% tab 'Scala 3 only' for=method_24 %}
```

Comment:
there should also be a tab with scala 2 equivalent

Reply:
changed to Scala 3 Only.
I cannot write scala 2 code.